### DISCUSSION TITLE
Are there any plans to add 4-way rail blueprints?

### DISCUSSION THREAD

**Discussion by CleverWild** [Opinionated-Blueprints/10-Books-Full-of-Rails#36] [Ideas]
After the release of the dlc, I'm not sure if these rails are needed, as elevated rails were added which greatly increased capacity. Also, based on your binding grid size, I don't think adding these rails would be easy, or even possible.
What do you think about this?

**Comment by Bocian-1** [comment]
The grid was barely large enough to do 4 lane. With wider curves, it's not possible anymore. 4 lane was always dumb either way. I made it when I was still fairly new to this so it stuck, but room on straight rail was never he throughput limiting factor. Intersections were and still are, and 4 lane intersections don't benefit much over 2 lane of the same size.
